compiler/noirc_driver/src/lib.rs

```diff
@@ -61,6 +61,10 @@ pub struct CompileOptions {
     /// Suppress warnings
     #[arg(long, conflicts_with = "deny_warnings")]
     pub silence_warnings: bool,
+
+    /// Enables the aztec macro preprocessor
+    #[arg(long, hide = true)]
+    pub aztec_macro: bool,
```

Comment:
Why hide the argument? I don't think we should hide any arguments, even `--show-ssa` and other already hidden arguments.

Reply:
I think this is because we don't want users to use these, only the aztec sandbox users but it will be done via the aztec-cli which will prepend the flag